### DISCUSSION TITLE
is there any directives like @layer for tailwindcss?

### DISCUSSION THREAD

**Discussion by sharmapukar217** [unocss/unocss#1815] [Q&A]
There is the support for layers as shown in documentation but is there any `@layer` directive so that it can be used inside css file?

**Comment by texastoland** [comment]
I found 3 options in incresing specificity:

1. [`@unocss`](https://unocss.dev/integrations/postcss#unocss) in CSS ([example](https://github.com/unocss/unocss/blob/main/examples/vite-vue3-postcss/src/style.css))
2. [`layer`](https://unocss.dev/config/layers) in config
3. [`layer-`](https://unocss.dev/presets/mini#css-layer) class names
